modules/dag-l0/src/main/scala/io/constellationnetwork/dag/l0/config/DelegatedRewardsConfigProvider.scala

```diff
@@ -91,13 +91,25 @@ object DefaultDelegatedRewardsConfigProvider extends DelegatedRewardsConfigProvi
         delegatorsWeight = NonNegFraction.unsafeFrom(50L, 100L)
       )
 
-  private val testnetDistributionProgram: EpochProgress => ProgramsDistributionConfig =
-    _ =>
+  private val testnetDistributionProgram: EpochProgress => ProgramsDistributionConfig = {
+    case epoch if epoch < EpochProgress(1027000) =>
```

Comment:
I know this is a magic number, but can we annotate this change with a comment indicating why this was chosen? Would be helpful in the future.

Reply:
we have never commented these epoch progress magic numbers before
https://github.com/Constellation-Labs/tessellation/blob/main/modules/dag-l0/src/main/scala/org/tessellation/dag/l0/config/types.scala#L113

how would you like to go about labeling them?

Reply:
Hmm, maybe a date would be easiest? Or a reference to a commit for when a particular change was introduced? I don't think you need to do this as part of this PR, just noting this for future discussion.